### PR TITLE
aktu.ac.in

### PR DESCRIPTION
**Pull request by gork7777**
issue: #79712

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -98,6 +98,7 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+aktu.ac.in#$#body { overflow: visible!important; }
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:101]: Scrolling will be broken with AG for iOS and AG Content Blocker, all 3 rules should be CSS-rules:
```
aktu.ac.in#$#body { overflow: visible!important; }
aktu.ac.in#$##onload { display: none!important; }
aktu.ac.in#$#.modal-backdrop { display: none!important; }
```

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/popups.txt:101]: Should I add the twitter widget to social filters?

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:101]: Yes, just add the domain to 2 rules under this comment:
`! ##.twitter-timeline + widgets.js`

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt:101]: @gork7777 
Please add to this rule too: `##.twitter-timeline`

**Pull request merged**